### ISSUE TITLE
[QUESTION] Could I trigger click event on a single row?

### ISSUE THREAD

**Issue by ilyablbnv** [glittershark/reactable#372]
So I fetch my JSON data from the server and use reactable library to display as a table. Could I trigger a click event on a single row and pass _id ? Previously I used a traditional _map loop so is there a possibility ? Thanks in advance.

**Comment by andrewlorenz** [comment]
Yes, this table seems fully featured for *displaying* data, but not for *manipulating* it!  There appears to be not a jot of support for anything that would allow you to have a (say), edit or delete button against each row.  What an utterly huge shame it doesn't.

**Comment by ssorallen** [comment]
You can add an `onClick` callback to the `<Tr>` component like you'd expect:

```
{rows.map(_id =>
  <Tr onClick={this.handleTrClick.bind(this, _id)}/>
)}
```

**Comment by kdzapp** [comment]
Doesn't work for me
